src/HelpText/HelpTextAttr.xml

```diff
@@ -53,6 +53,11 @@
             <category>Common</category>
             <description>Maximum height of the help box in pixels</description>
         </property>
+        <property key="renderHTML" type="boolean" defaultValue="false">
+            <caption>Render as HTML</caption>
+            <category>Behavior</category>
+            <description>Render as HTML. Be aware that this could lead to XSS attacks, where user input is rendered in the user's browser.</description>
```

Comment:
I am typing in a text input box. It is rendered in my browser. Am I performing an XSS attack now?